Lib/argparse.py

```diff
@@ -1000,8 +1010,10 @@ def __init__(self,
             help=help)
 
     def __call__(self, parser, namespace, values, option_string=None):
-        new_count = _ensure_value(namespace, self.dest, 0) + 1
-        setattr(namespace, self.dest, new_count)
+        count = getattr(namespace, self.dest, None)
+        if count is None:
+            count = 0
```

Comment:
Why not just use getattr(namespace, self.dest, 0) and not have the if test?

Reply:
If think the current code is equivalent to `getattr(namespace, self.dest) or 0`, not `getattr(namespace, self.dest, 0)`.  The first one can never have None for count, the second does (if namespace has an attribute with the name in self.dest and the value None).